source/packages/freeradius-package.rst

```diff
@@ -0,0 +1,152 @@
+FreeRADIUS package
+==================
+
+Basics
+------
+
+Installation
+^^^^^^^^^^^^
+
+-  Navigate to **System > Packages**, **Available Packages** tab
+-  Click |fa-plus| at the end of the row for **freeradius3**
+-  Confirm the installation
+-  Monitor the progress as it installs
+
+After Installation, the service may be configured at **Services >
+FreeRADIUS**.
+
+First Configuration
+^^^^^^^^^^^^^^^^^^^
+
+-  Select the **interface(s)** on which the RADIUS server should listen
+   on.
+-  Configure the **NAS/client(s)** from which the RADIUS server should
+   accept packets.
+-  Add the **user(s)** who should have access.
+
+After this, have a look at the pfSense syslog. There should be the
+following::
+
+  radiusd[16634]: Ready to process requests.
+  radiusd[16627]: Loaded virtual server
+
+
+Troubleshooting RADIUS Authentication
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When attempting to authenticate against a RADIUS server, errors may be
+encountered in the logs that prevent it from working properly. Here are some
+errors and how to resolve them::
+
+  mpd: [pt0] RADIUS: RadiusSendRequest: rad_init_send_request failed: -1
+
+* This appears to happen when the RADIUS shared secret contains special
+  characters. Try again with an alphanumeric shared secret.
+
+
+General Information
+-------------------
+
+Features
+^^^^^^^^
+
+The features below were tested on pfSense
+
+- Authentication with Captive-Portal
+- Pre-defined user attributes and custom check-items and reply-items
+- NAS/Clients running on IPv4 and IPv6
+- Interfaces can listen on IPv4 and IPv6
+- OpenVPN + Username + RADIUS and OpenVPN + Username + Cert + RADIUS
+- Auth with PAP, CHAP, MSCHAP, MSCHAPv2
+- Auth with EAP-MD5 + dynamic VLAN assignment
+- Auth with PEAP + dynamic VLAN assignment
+- Auth with EAP-TLS/EAP-TTLS + dynamic VLAN assignment
+
+  .. code::
+
+    radiusd[3206]: Login OK: [testuser/<via Auth-Type = EAP>] (from client pfsense port 0 cli 00-04-23-5C-9D-19)
+    radiusd[3206]: Login OK: [testuser/<via Auth-Type = EAP>] (from client pfsense port 0 cli 00-04-23-5C-9D-19)
+    radiusd[3206]: Login OK: [testuser/<via Auth-Type = EAP>] (from client pfsense port 0 via TLS tunnel)
+    radiusd[3206]: Login OK: [testuser/<via Auth-Type = EAP>] (from client pfsense port 0 via TLS tunnel)
+
+- Simultaneous-Use - The following will be present in the system log::
+
+    radiusd[3206]: Multiple logins (max 1) : [testuser/testpw] (from client testing port 10)
+
+- A certain amount of time per day/week/month/forever (CHECK-ITEM:
```

Comment:
`CHECK-ITEM: Max-Daily-Session := 60` should be escaped, e.g. ` ``CHECK-ITEM: Max-Daily-Session := 60`` `

Reply:
I'll update this. However I didn't make change to this file (I simply renamed `usermanager/freeradius-2-x-package.rst` to `source/packages/freeradius-package.rst`).